packages/react/src/MinimalFooter/MinimalFooter.module.css

```diff
@@ -0,0 +1,110 @@
+.Footer {
+  background-color: var(--base-color-scale-gray-9);
+}
+
+/**
+ * Footer shared styles
+ */
+.Footer__container {
+  max-width: var(--brand-footer-max-width, 1280px);
+  padding: 0 28px; /* should match SubdomainNavBar */
+  margin-right: auto;
+  margin-left: auto;
+}
+
+/**
+ * Footer terms section
+ */
+.Footer__terms {
+  padding: var(--base-size-48) 0;
+}
+
+.Footer__terms-item {
+  margin-bottom: var(--base-size-24);
+}
+
+.Footer__terms-item sup {
+  font-size: var(--base-size-12);
+  vertical-align: super;
+  display: inline-block;
+  margin-right: var(--base-size-4);
+}
+
+.Footer__terms-item:last-of-type {
+  margin-bottom: 0;
+}
+
+/**
+ * Footer social and logomarks section
+ */
+.Footer__social-links {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  justify-content: flex-start;
+  align-items: center;
+  gap: var(--base-size-16);
+}
+
+.Footer__logomarks {
+  border-top: var(--brand-borderWidth-thin) solid var(--brand-color-border-default);
+  border-bottom: var(--brand-borderWidth-thin) solid var(--brand-color-border-default);
+}
+
+.Footer__logomarks,
+.Footer__legal-and-links {
+  padding-top: var(--base-size-48);
+  padding-bottom: var(--base-size-48);
+}
+
+.Footer__social-label {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  word-wrap: normal;
+  border: 0;
+}
+
+/**
+ * Footer legal section
+ */
+
+.Footer__copyright {
+  display: flex;
+  order: 2;
+  padding-bottom: var(--base-size-4);
+}
+
+@media screen and (min-width: 48rem) {
+  .Footer__copyright {
+    order: 1;
+  }
+}
+
+.Footer__links {
+  order: 1;
+}
+
+@media screen and (min-width: 48rem) {
+  .Footer__links {
+    order: 2;
+  }
+}
```

Comment:
I believe using `order` might break the logical source order and make this non-conforming from an accessibility standpoint. Could we achieve a similar design without relying on the `order` property?

Reply:
You could use grid which maintains the semantic order while affecting layout if you wanted. I think grid is becoming stronger than flex for this reason.

Reply:
We can keep `order` for now and work with the accessibility team on this one.

Reply:
> You could use grid which maintains the semantic order while affecting layout if you wanted. 

This would unfortunately have the same issue as flex order, where the change is only visual.

Two options to resolve this:

- Use a JS media query to move the semantic order based on client window size (expensive)
- Switch the copyright to appear on the right (different to design)

Given none of these are ideal, @danielguillan and I have agreed to keep it as-is for now and revise in future iteration after discussing with a11y team.